scripts/tvm_cli/Dockerfile.dependencies.arm64

```
@@ -0,0 +1,48 @@
+ARG BASE_IMAGE
+FROM $BASE_IMAGE
```

Testing this on `arm64` hardware revelaed that this is different from the `amd64` file, causing this variable to be blank when `docker build` is called from the build script. I believe this should be:

```suggestion
ARG FROM_ARG
FROM $FROM_ARG
```

Or the build script should be modified to use `BASE_IMAGE` instead and the `amd64` Dockerfile modified to match.